packages/wallets/src/features/cashier/modules/TransactionStatus/TransactionStatus.scss

```diff
@@ -0,0 +1,43 @@
+.transaction-status {
+    &-container {
+        position: absolute;
+        top: 0;
+        right: 0;
+        width: 28.2rem;
+        display: flex;
+        align-self: stretch;
+        flex-direction: column;
+        align-items: flex-start;
+        gap: 0.8rem;
+        background-color: var(--general-section-1);
+        border-radius: $BORDER_RADIUS * 2;
```

Comment:
BORDER_RADIUS is not defined. Use normal numbers for now

Reply:
Done